server.ts

```diff
@@ -1,3 +1,6 @@
+// Require app insights before anything else to allow for instrumentation of bunyan and express
+import 'applicationinsights'
```

Comment:
not sure this is strictly necessary but won't break anything

Reply:
I did wonder why we didn't have it in the first place, but thought maybe it was something ~ mysterious ~ that happened once you imported haha!